setup.py

```diff
@@ -51,9 +51,9 @@
     'Pygments>=2.0',
     'docutils>=0.11',
     'snowballstemmer>=1.1',
-    'babel',
-    'alabaster',
-    'sphinx_rtd_theme',
+    'babel>=1.3',
+    'alabaster>=0.7,<0.8',
+    'sphinx_rtd_theme>=0.1,<0.2',
```

Comment:
@ericholscher do you have a plan to release a new sphinx_rtd_theme in a day? Or do you have any opinion for about the version range `'sphinx_rtd_theme>=0.1,<0.2'`?